lmfdb/bianchi_modular_forms/bianchi_modular_form.py

```diff
@@ -283,8 +283,7 @@ def bmf_field_dim_table(**args):
     weights = set()
     for dat in data:
         weights = weights.union(set(dat[gl_or_sl].keys()))
-    weights = [int(w) for w in weights]
-    weights.sort()
+    weights = sorted([int(w) for w in weights])
```

Comment:
I probably agree with this change for readability, though I'll note that the old version is slightly faster unless Python is doing some kind of optimization.